### ISSUE TITLE
Uninstall String is incorrect in registry

### ISSUE THREAD

**Issue by PowershellScripter** [d2phap/ImageGlass#931]
### System information:
 - Windows OS version: Windows 10 2009
 - ImageGlass version: 8.0.12.8

### To Reproduce
1. 
2. 
3. 
4. 

### Actual behavior:
Is stated as msiexec.exe /I {GUID}

### Expected behavior:
Should be stated as msiexec.exe /X {GUID}

### Screenshots:
![issuewithuninstall](https://user-images.githubusercontent.com/71091773/101831658-76d3fb80-3af3-11eb-9d01-4b0c4acd4ce8.jpg)

**Comment by d2phap** [comment]
Hi @NinjaRMMTheming 

I'm using Advanced Installer to create the MSI file, I'm not sure what's the problem is.
You can help to check the source code:
https://github.com/d2phap/ImageGlass/blob/develop/Setup/AdvancedInstaller/x64/ImageGlass-x64.aip

**Comment by PowershellScripter** [comment]
@d2phap

Still trying to figure this part out. I combed through the advanced installer attributes but havent found anything yet. I reached out to AI but they havent gotten back with me. 

If I understand the installer correctly, there is a registry section that allows you to set custom keys to be created. I noticed no actual reg keys exists in your aip file.

Also, there is another bug that im going to generate about the admin config

**Comment by PowershellScripter** [comment]
@d2phap

I checked with AI and they said the registry key is set based on the MSI installer. Meaning the built MSI package of ImageGlass is what sets the registry key for ImageGlasses uninstall string.

So I checked forums and here is what I found:
 "The answer is to set the ARPNOMODIFY property in the MSI to 1 to generate an UninstallString that uses the /X parameter.
The /I param means that you have the "Change" and "Repair" options available in appwiz.cpl listing. "

So, if you decide to leave the uninstallstring as is, it gives users the ability to 'Change' or 'Repair' from Programs and Features. 

If you WANT to change it to be /X as the switch, you apparently need to chnage the builder value of  ARPNOMODIFY property in the MSI to 1.

Either way, I am instead just telling my scripts to pull the product code and then run MSIexec /X {ProductCode} to uninstall.

Just odd that previous versions of ImageGlass didnt have this issue.
